content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -1,3 +1,9 @@
+---
+title: Installation Guide using Ansible
+description: "This document describes how to install SODA projects in a local cluster with detailed configuraiton options. These steps will help you to enable / disable projects. After installation using these steps, you can get the features of all the enabled projects. You can test either through SODA Dashboard UI or CLI"
```

Comment:
This text is duplicated

Reply:
This description is not visible on the page. It is the front matter and this description is used to display the description on the Getting Started page.

Reply:
ok